### ISSUE TITLE
Can i use fine tuning and data augmentation together?

### ISSUE THREAD

**Issue by zishanahmed08** [MarvinTeichmann/KittiSeg#70]
In intend to fine tune to fine tune the Kitti_seg_pretrained model turning data augmenatation on. Is this a good idea? The intention to retrain on Kitti Dataset with augmentation on so that it can segment better on non kitti images.

Can I fine tune by following the below setps  in KittiSeg\RUNS\KittiSeg_pretrained\model_files\...Did i miss any steps?

1)augment_level = 2 
2)whitening to true
3)max_steps to greater than 16000

Thank you Marvin for your wonderful work!

**Comment by MarvinTeichmann** [comment]
Yes, you can use fine-tuning and augmentation together. However I would suggest starting of with `augment_level = 1`. augment_level = 2 seams to hurt performance. Also, don't set whitening to true.  Whitening is already done in the architecture file.
